wasp/src/main/java/com/orhanobut/wasp/parsers/GsonParser.java

```diff
@@ -22,7 +22,7 @@ public GsonParser(Gson gson) {
     }
 
     @Override
-    public <T> T fromJson(String content, Type type) {
+    public <T> T fromJson(String content, Type type) throws Exception{
```

Comment:
Instead of general exception. throwing JsonSyntaxException will be better in order to analyze the error clearly. Gson throws this exception.